reference/lib/ReferenceOrderCombiner.sol

```diff
@@ -481,8 +481,10 @@ contract ReferenceOrderCombiner is
 
             // If offerer and recipient on the execution are the same...
             if (execution.item.recipient == execution.offerer) {
-                // increment total filtered executions.
-                totalFilteredExecutions += 1;
+                unchecked {
```

Comment:
Can you leave a comment as to why unchecked is safe here above the block and others? (i.e. that it is infeasible to increment that high)

Reply:
@0age Done.